content/reference-currency.md

```diff
@@ -0,0 +1,60 @@
+# Currency Formats
+
+The XRP Ledger has [two kinds of money](concept-money.html): XRP, and issued currencies. In the XRP Ledger, both types have high precision, although their formats are different.
+
+## String Formatting
+
+XRP Ledger APIs generally use strings, rather than native JSON numbers, to represent numeric amounts of currency for both XRP and issued currencies. This protects against a loss of precision when using JSON parsers, which may automatically try to represent all JSON numbers in a floating-point format.
+
+## XRP Precision
+
+XRP has the same precision as a 64-bit unsigned integer where each unit is equivalent to 0.000001 XRP. Its properties are:
+
+* Minimum value: `0`
+* Maximum value: `100000000000` (10^11) XRP
```

Comment:
Can you insert arbitrary HTML? If so, maybe `10<sup>11</sup>` would look better.